gulpfile.js/build.js

```diff
@@ -528,23 +535,23 @@ function collectStatics(done) {
  *
  * @return {undefined}
  */
-function persistBuildInfo(done) {
-  Promise.all([git.user(), git.version(), git.message()])
-    .then(([user, version, message]) => {
-      const buildInfo = {
-        'number': travis.build.number || null,
-        'at': new Date(),
-        'by': user,
-        'environment': config.environment,
-        'commit': {
-          'sha': version,
-          'message': message,
-        },
-      };
-
-      fs.writeFile(project.paths.BUILD_INFO, yaml.safeDump(buildInfo), done);
-    })
-    .catch((error) => done(error));
+async function persistBuildInfo() {
+  const version = await git.version();
+  const user = await git.user();
+  const message = await git.message();
+
+  const buildInfo = {
+    'number': travis.build.number || null,
```

Comment:
when would `travis.build.number` not be defined? If it isn't, are we sure `travis.build` exists?

Reply:
ping @bezoerb

Reply:
Yep, `travis.build` (and even `.number`) will always exist though may be undefined:

https://github.com/ampproject/amp.dev/blob/4f010c325688cf2c1037f2e56d1b0d3ed159fd4e/platform/lib/utils/travis.js#L51-L54